testing/bcc/APKBUILD

```diff
@@ -33,6 +34,28 @@ build() {
 package() {
 	cd "$builddir/build"
 	make install DESTDIR="$pkgdir"
+
+	mv "$pkgdir"/usr/share/bcc/man "$pkgdir"/usr/share/man
```

Comment:
Tools which these man pages refer to are not in PATH, so it's better to leave them in `/usr/share/bcc/man`.
Furthermore in a shared space they would conflict with man pages provided by bpftrace